pinc/mentorbanner.inc

```diff
@@ -33,21 +33,17 @@ function mentor_banner($round)
         case 0:
         case 1:
         case 2:
-            $font_boost = "1em";
-            $font_col = '#339933';
+            $class = 'mentor-recent';
             break;
         case 3:
         case 4:
-            $font_boost = "large";
-            $font_col = "#FF6600";
+            $class = 'mentor-older';
             break;
         default:
-            $font_boost = "x-large";
-            $font_col = "#FF0000";
+            $class = 'mentor-oldest';
             break;
     }
-
-    echo "<p style='color: $font_col; font-weight: bold; font-size: $font_boost; text-align: center'>";
+    echo "<p class = '$class'>";
```

Comment:
We don't include spaces between tag params and their values, so let's remove the extra spaces here.